modules/@apostrophecms/widget-type/index.js

```diff
@@ -422,13 +445,14 @@ module.exports = {
         return [];
       },
 
-      addWidgetOperation(operation) {
-        self.apos.area.addWidgetOperation({
-          ...operation,
-          type: self.__meta.name
+      checkWidgetOperationsPermissions(req) {
```

Comment:
we should rename this function to reflect what it does in my opinion. It gets the widget operations for the current user.

Reply:
Renamed